### PR TITLE
Modpacks vs. Distribution

### PR DESCRIPTION
**Pull request by Strikingwolf**

### CODE REVIEW

**Comment by Strikingwolf** [comment]
@jakimfett I have fixed this pr to work with @cuchaz's commits. Look through it and see if there is anything you want to change and if it is good then please merge it before you merge others :laughing:

File: LICENSE.md [hunk 1]
```diff
@@ -67,24 +70,37 @@ The user has the right to decompile the binaries and examine the resulting sourc
...
 7. Right to distribute addons
 -----------------------------
 
-Ths user is granted the right to distribute addons to this mod, which may be distributed under a different license, as long as the addon is not a derivative or it includes only the API of this mod. For example, if an addon extended this mod using the mod's API, but did not include other derived code, then it can be distributed under another license.
+The user is granted the right to distribute addons to this mod, which may be distributed under a different license, as long as the addon is not a derivative or it includes only the API of this mod. For example, if an addon extended this mod using the mod's API, but did not include other derived code, then it can be distributed under another license.
+
+8. Right to distribute in a modpack
+-----------------------------------
+
+Using the link the author provides is not required when distributing mods in a modpack. However, modpacks may only be hosted on sites/launchers given permission by the original mod author. Modpacks may not be used to generate revenue.
```
Comment (jakimfett) [inline LICENSE.md]: I kinda want to change this a bit...I'd prefer that the license grant modpack permissions by default, but give the author the ability to revoke permissions on a case by case basis.

**Comment by jakimfett** [comment]
Looks good overall. Minor tweaks needed, but I'll go ahead and merge this.

**Comment by cuchaz** [comment]
I might be wrong, but I think the new modpack distribution section conflicts with the distribution of unmodified copies section.

**Comment by Strikingwolf** [comment]
@cuchaz it does, it is more like a modpack clause. Modpacks are kinda've special in a way because they aren't downloading each mod on the fly.

**Pull request merged**